### ISSUE TITLE
Container Service API 2018-03-01

### ISSUE THREAD

**Issue by tombuildsstuff** [Azure/azure-sdk-for-go#2069]
👋 

I noticed that version 2018-03-01 is now available in the latest branch: https://github.com/Azure/azure-sdk-for-go/tree/fa8ed08ed42917b9f3a3f27f1096ec92a81de65c/services/containerservice/mgmt/2018-03-31/containerservice

Since it's a new/additive SDK version - would it be possible to include this in the next SDK release? We're looking to upgrade to it to be able to unblock a few issues for us:

- https://github.com/terraform-providers/terraform-provider-azurerm/issues/1389
- https://github.com/terraform-providers/terraform-provider-azurerm/issues/1276
- and I believe it may impact this issue: https://github.com/terraform-providers/terraform-provider-azurerm/issues/929

Thanks!

**Comment by vladbarosan** [comment]
Hi @tombuildsstuff we have it already in the latest branch if you want to try it out [here](https://github.com/Azure/azure-sdk-for-go/tree/latest/services/containerservice/mgmt/2018-03-31/containerservice)

We will be looking to release 17.4 soon which will include `Container Service API 2018-03-01`.

**Comment by koalalorenzo** [comment]
Any ETA on this? :) When this will be ready? 
It would be really nice to have AAD + RBAC in Terraform: https://github.com/terraform-providers/terraform-provider-azurerm/issues/1389

**Comment by jhendrixMSFT** [comment]
v17.4.0 is now available https://github.com/Azure/azure-sdk-for-go/releases/tag/v17.4.0

**Comment by koalalorenzo** [comment]
@jhendrixMSFT what about AAD set up? Is there an issue already for this? (I can't see anything useful into the changelog... )

**Comment by jhendrixMSFT** [comment]
@koalalorenzo I'm not sure what you mean by this part, can you please clarify?  If it's something separate from adding the new container service API version we should track it in a separate issue.

**Comment by koalalorenzo** [comment]
Since AKS went GA with new features, it would be nice to know if there is already the possibility to use the API to create a new AKS cluster with these new features (one of those is Azure Active Directory + RBAC).

Is this a possibility? Is there already an API to deploy a kubernetes cluster with Azure Active Directory enabled?

**Comment by jhendrixMSFT** [comment]
@joshgav do you know, or know who we can follow up with about this?

**Comment by joshgav** [comment]
Opened a new issue to discuss further: https://github.com/Azure/azure-sdk-for-go/issues/2089

**Comment by vladbarosan** [comment]
@koalalorenzo please follow up on the seperate issue https://github.com/Azure/azure-sdk-for-go/issues/2089 that @joshgav  referenced.
